src/t8_netcdf/t8_nc_mesh.hxx

```diff
@@ -9,6 +9,28 @@
 #include <t8_netcdf/t8_nc_hyperslab.hxx>
 
 #include <utility>
+#include <numeric>
+
+/*Function to calculate all divisors of a number, sorted from the smallest to the biggest.*/
+std::vector<int> divisors(t8_gloidx_t n) {
```

Comment:
Maybe we could move the definition of several of those functions into "t8_nc_mesh.cxx" instead of defining them in the header as inline.